packages/terra-clinical-result/src/terra-dev-site/doc/example/Examples.module.scss

```diff
@@ -0,0 +1,95 @@
+:local {
```

Comment:
Why are adding custom styles for our documentation examples? Do these components work standalone?